examples/mistral-e2e/examples/chat.py

```diff
@@ -0,0 +1,60 @@
+from transformers import AutoTokenizer
+from optimum.neuron import NeuronModelForCausalLM
+
+model = NeuronModelForCausalLM.from_pretrained("./mistral_neuron", local_files_only=True)
```

Comment:
How `./mistral_neuron` is downloaded?

Reply:
mistral_neuron is outputted from the compiler, the idea is to transfer the fine-tuned model from the Trainium instance to Inferentia via S3.